### ISSUE TITLE
Can't get rid of it.

### ISSUE THREAD

**Issue by ghost** [jshackles/RetroGOG#18]
This was a cool idea at the time.  All my emulators in one place.  But the time it took for Galaxy to import all of those games every start up was just plain annoying.  

So I tried to uninstall RetroGog by removing all the apis.  Didn't work.  Uninstalled/reinstalled Galaxy.  Didn't work.  Did a 100% clean install, ie, removed all the appdata, programdata, ect.  Didn't work.  I think I broke Galaxy.  The integrations are gone, but the games remain.  I can't get rid of them.  Even more annoying, they take priority over everything else in Galaxy.  It will wait to load all of my Retroarch ROMs before it gives me access to my GOG games.  Also, it seems to be interfering with adding new games I bought on GOG, they're not showing up in Galaxy.

It's madness.

**Comment by Pab10Suarez** [comment]
ok soo um to remove the games you need to first disconnect your integration and then you can remove the integration manually
you could try to reconnect your emulator integrations, then disconnect so it can remove games properly

**Comment by ghost** [comment]
Hey.  Thanks for being so quick to reply.  It's worth a try.

**Comment by ghost** [comment]
It worked!  Thanks.

To close this out.  I really love the idea of this...but it's just not there yet.  It's an early version (beta?) being added to a program that is already in beta.  Given some time, I think it'll be great.  But I'm going to stay away for while before trying again.

**Comment by Pab10Suarez** [comment]
> It worked! Thanks.
> 
> To close this out. I really love the idea of this...but it's just not there yet. It's an early version (beta?) being added to a program that is already in beta. Given some time, I think it'll be great. But I'm going to stay away for while before trying again.

yeah i understand you it needs more time the problem is we don't know ho much it will take, i hope someday we'll be able to see achievements in galaxy interface taht would be so cool, if i remember i'll let you know when gog or the integration creator fix the thing with games installed i know it can be annoying for me it's not because i study in my computer in the day so i don't have to wait the process that takes like one hour

**Comment by Pab10Suarez** [comment]
Heya, i just reported the bug and it seems they will fix it in the next update https://github.com/gogcom/galaxy-integrations-python-api/issues/180#issuecomment-864614490

**Comment by DreadfulDrummer** [comment]
> Heya, i just reported the bug and it seems they will fix it in the next update [gogcom/galaxy-integrations-python-api#180 (comment)](https://github.com/gogcom/galaxy-integrations-python-api/issues/180#issuecomment-864614490)

In an "upcoming release" not necessarily the next one

**Comment by Pab10Suarez** [comment]
> > Heya, i just reported the bug and it seems they will fix it in the next update [gogcom/galaxy-integrations-python-api#180 (comment)](https://github.com/gogcom/galaxy-integrations-python-api/issues/180#issuecomment-864614490)
> 
> In an "upcoming release" not necessarily the next one

well it was the next update since that comment but anyway, it's  fixed now at least for me, 
you could check it out, @redbeard1084

**Comment by DreadfulDrummer** [comment]
Yep, been fixed a short while now. Library is populated upon opening, all RetroArch games ready to go (which was my problem).

**Comment by ghost** [comment]
Hey.  Thanks for the tip.  I'll check it out.

**Comment by ghost** [comment]
I'm back and in even more trouble!  I decided to give the new update a try.  I decided to just integrate my SNES ROMs.  I can not get them out of the library no matter what I do.  This new version slogs GOG to an agonizing crawl.  It literally takes nearly an hour to import the library and crashes when I close the import.  I did several reinstalls.  It doesn't recognize the SNES as being integrated so how can I take it out of integration.  I even went so far as to reinstall retroarch and try to re-run retrogog's install wizard with an empty library.  Now it gives me an "unexpected error message" whenever I try to run it.  This is an absolute mess and do not speak with hyperbole when I say it has completely destroyed my entire setup for how I play games on this PC.  Please help me get rid of this problem so I can walk away a much sadder but wiser man and never try anything like this ever again.

**Comment by ineeddspelchek** [comment]
> I'm back and in even more trouble! I decided to give the new update a try. I decided to just integrate my SNES ROMs. I can not get them out of the library no matter what I do. This new version slogs GOG to an agonizing crawl. It literally takes nearly an hour to import the library and crashes when I close the import. I did several reinstalls. It doesn't recognize the SNES as being integrated so how can I take it out of integration. I even went so far as to reinstall retroarch and try to re-run retrogog's install wizard with an empty library. Now it gives me an "unexpected error message" whenever I try to run it. This is an absolute mess and do not speak with hyperbole when I say it has completely destroyed my entire setup for how I play games on this PC. Please help me get rid of this problem so I can walk away a much sadder but wiser man and never try anything like this ever again.

I had the same issue or maybe just similar. When you reinstalled this plugin to try and take out the integration, did you still have your SNES titles set up in Retroarch or had you gotten rid of them?

**Comment by JokeDeity** [comment]
I'm not able to re-add the integrations because they never successfully added in the first place and I still cannot properly get this to install, but the GAMES added to my games list.  So how in the actual flying fuck do I get rid of them if I cannot in any way get the integrations to show up in my settings?  This is the worst fucking thing I've ever installed and I've spent countless hours on this.

**Comment by chuckzee** [comment]
@JokeDeity wipe the whole GOG install and be done with it, start fresh and do the work yourself instead of berating people.

**Comment by JokeDeity** [comment]
What a dickhead.  Go be useless somewhere else.  I've already reinstalled it.

**Comment by chuckzee** [comment]
> What a dickhead. Go be useless somewhere else.

Great advice, you should take it.
